### ISSUE TITLE
TextInput's suggestions are not readable by screen readers and voice over tools. 

### ISSUE THREAD

**Issue by jlevesque** [grommet/grommet#1759]
The suggestion item in the list doesn't have an aria-label. The voice over tools returns a [Object object] item since they are unable to read it. Adding an aria-label on the li should made it more accessible.

### Expected Behavior
The screen readers and voice over tools should be able to read the suggestions.

### Actual Behavior
The screen readers and voice over tools reads the suggestion as a [Object object] item.

### URL, screen shot, or Codepen exhibiting the issue

```
_renderDropContent () {
    const { suggestions } = this.props;
    const { activeSuggestionIndex } = this.state;
    let items;
    if (suggestions) {
      items = suggestions.map((suggestion, index) => {
        let classes = classnames(
          {
            [`${CLASS_ROOT}__suggestion`]: true,
            [`${CLASS_ROOT}__suggestion--active`]:
              index === activeSuggestionIndex
          }
        );
        return (
          <li key={index} className={classes}
            onClick={this._onClickSuggestion.bind(this, suggestion)}>
            {this._renderLabel(suggestion)}
          </li>
        );
      });
    }
```

### Your Environment

* Grommet version:1.9.0
